packages/indexer-client/src/types/serverTypes.ts

```diff
@@ -149,6 +150,12 @@ export interface IndexerServerClaimArbMerkleProofsParams {
   address: string;
 }
 
+export interface IndexerServerMakerStatisticsParams {
+  productId: number;
```

Comment:
Seems like for the server it'd be snake case and not camelcase?

Reply:
Ah yea good catch